README.md

```diff
@@ -10,4 +10,61 @@ I will not be guarenting anything. I'll try to target both bluesky & mastodon, b
 
 ### This is not (and probably won't be for a while) a 100% accurate recreation of the twitter API
 
-most of this readme todo
\ No newline at end of file
+most of this readme todo
+
+## Compatibility
+**iOS:** 4.2.1 works
```

Comment:
The iOS version number isn't what actually matters for running twitter. We specifically care about the twitter version. I have been testing on Twitter 4.1.3. (also you might want to note that this currently should work with client from ~2010-2012)

Reply:
Now looking at it, it looks like i did write the twtter version as 4.2.1 up in the readme. oops.